src/elements/fields/PinInput.jsx

```diff
@@ -186,9 +192,11 @@ function OtpInput({ onChange, onClick, value, field, inlineError }) {
                     onChange={handleOnChange}
                     onKeyDown={handleOnKeyDown}
                     onInput={handleOnInput}
+                    paste={pasted}
                     onPaste={handleOnPaste}
                     onFocus={(e) => {
```

Comment:
what if the user starts typing again after pasting but before re-focusing and setting paste back to false?

Reply:
That's fine, it'll go back to autohighlighting so they can keep typing (and overwriting if need be)

Reply:
correction: it will do that now, your comment made me go looking for the edge case and it's handled now, thanks!